online-bookstore/src/main/java/group1/itschool/onlinebookstore/models/dto/BookDTO.java

```diff
@@ -5,6 +5,20 @@
 @Data
 public class BookDTO {
 
-    private String name;
-    private String category;
+    private String title;
+    private String author;
+    private String publisher;
+    private int publicationDate;
+    private int codeISBN;
+    private String genre;
```

Comment:
In general all the fields should have spring validation. By validating at the field level, you can catch errors early in the processing pipeline, which can help improve the overall performance and efficiency of your application. Will stop at this field however all the fields should have validation on the BookDTO